deltacat/tests/compute/compactor_v2/test_compaction_session.py

```diff
@@ -176,7 +176,7 @@ def test_compact_partition_when_rcf_was_written_by_past_commit(
                         **local_deltacat_storage_kwargs,
                         **{"equivalent_table_types": []},
                     },
-                    "primary_keys": [],
+                    "primary_keys": ["pk"],
```

Comment:
pk doesn't matter because hash_bucket_count = 1. Why was this test passing before?